stackit/internal/services/argus/scrapeconfig/resource.go

```diff
@@ -758,6 +1408,99 @@ func toCreatePayload(ctx context.Context, model *Model, saml2Model *saml2Model,
 	}
 	sc.StaticConfigs = &t
 
+	if sc.TlsConfig == nil && !tlsConfigModel.InsecureSkipVerify.IsNull() && !tlsConfigModel.InsecureSkipVerify.IsNull() {
+		sc.TlsConfig = &argus.CreateScrapeConfigPayloadHttpSdConfigsInnerOauth2TlsConfig{
+			InsecureSkipVerify: conversion.BoolValueToPointer(tlsConfigModel.InsecureSkipVerify),
+		}
+	}
+
+	mrcs := make([]argus.CreateScrapeConfigPayloadMetricsRelabelConfigsInner, len(metricsRelabelConfigsModel))
```

Comment:
We could probably improve these names `sc` is already not the best, but at least its the root level. Here we could call his `metricsRelabelConfigs`

Reply:
Renamed